fineract-provider/src/main/java/org/apache/fineract/infrastructure/hooks/processor/WebHookService.java

```diff
@@ -38,22 +38,22 @@ public interface WebHookService {
     String API_KEY_HEADER = "X-Fineract-API-Key";
 
     // Ping
-    @GET("/")
+    @GET(".")
```

Comment:
@daviestobialex why? "." is not necessary here...

Reply:
@vidakovic it is for the ping request to actually work the `.` has to be there. although I do not see where that `sendEmptyRequest` is being referenced.